### PR TITLE
Better phpstan default configuration

### PR DESCRIPTION
**Pull request by jdeniau**
- If a config file is found, then do not set the default level to `4`, the level should be in the configuration file (except if you force it of course).
- Mark phpstan error as error, not warning, as phpstan does not have warning, only errors.
- Use the `--no-progress` parameter

Thank you !

### CODE REVIEW

File: test/handler/test_phpstan_handler.vader [hunk 1]
```diff
@@ -68,10 +66,57 @@ Execute(Output for .inc files should be parsed correctly):
...
-  \     'type': 'W'
+  \     'type': 'E'
   \   }
   \ ],
   \ ale_linters#php#phpstan#Handle(bufnr(''), [
-  \   ' 1/1 [▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓] 100%',
   \   '/phpstan-test-files/foo/test.inc:9:Access to an undefined property Test::$var.',
   \])
+
+Execute(Test phpstan command output):
```
Comment (w0rp) [inline test/handler/test_phpstan_handler.vader]: Put these tests in a separate file in the `test/command_callback` directory. They may already be one. Use the special assertion commands documented in `:help ale-tests` instead.

File: ale_linters/php/phpstan.vim [hunk 2]
```diff
@@ -12,14 +12,25 @@ function! ale_linters#php#phpstan#GetCommand(buffer, version) abort
     \   ? ' -c ' . l:configuration
     \   : ''
 
+    let l:level =  ale#Var(a:buffer, 'php_phpstan_level')
+
+    if empty(l:level) && !filereadable('phpstan.neon')
```
Comment (w0rp) [inline ale_linters/php/phpstan.vim]: You should use the `ale#path#FindNearestFile` function used in other files to find this. This code will only work if the cwd for Vim contains this file.

Comment (jdeniau) [reply ale_linters/php/phpstan.vim]: You are right, thanks !

**Comment by jdeniau** [comment]
@w0rp changes made

**Comment by w0rp** [comment]
Cheers! :beers:

**Pull request merged**